### PR TITLE
Trying map

### PR DESCRIPTION
**Pull request by coleanthony1990**
# Pull Request

## Description:

- create a map view of each business rendered. 
- bringing in google maps api. 
- format phone to be clickable using href

## Next Steps:
format address?

### CODE REVIEW

File: src/components/SingleResultPage/SingleResultPage.js [hunk 1]
```diff
@@ -1,29 +1,53 @@
 import "./SingleResultPage.css";
-// import { data } from "../../assets/mock-data";
+import GoogleMapReact from 'google-map-react';
+import {ImLocation} from 'react-icons/im'
+import {AiOutlineStar} from 'react-icons/ai'
+const apiKey = process.env.REACT_APP_GOOGLE_API_KEY
```
Comment (vfields) [inline src/components/SingleResultPage/SingleResultPage.js:5]: So much fun cracking this together!

**Review by vfields — APPROVED** [review]
Completed alongside the FE crew, looks great!

File: src/components/SingleResultPage/SingleResultPage.js [hunk 2]
```diff
@@ -1,29 +1,53 @@
...
+          <p>{display_address.display_address}</p>
+          <a href={phone}>{display_phone}</a>
           <p>Check out {title}'s <a href={site} target="_blank" rel="noopener noreferrer">website</a>!</p>
         </div>
-        <img className="favorite-icon" src={placeholder} alt="an outline of a star" />
+        <AiOutlineStar className="favorite-icon"/>
       </article>
+      <div style={{ height: '50vh', width: '50%' }}>
+      <GoogleMapReact
+        bootstrapURLKeys={{ key: apiKey }}
```
Comment (Ant-Shell) [inline src/components/SingleResultPage/SingleResultPage.js:40]: Cool feature!

**Review by Ant-Shell — APPROVED** [review]
Reviewed with the team - approved!

**Pull request merged**